crypto/signature_collector.go

```diff
@@ -85,7 +85,8 @@ func (sc *SignatureCollector) Accept(
 			if err != nil {
 				return err
 			}
-			buffer, err = gomime.DecodeCharset(buffer, params)
+			mediaType, _, _ := mime.ParseMediaType(header.Get("Content-Type"))
```

Comment:
It wouldn't hurt checking `ParseMediaType` error also. Since
```
 If there is an error parsing the optional parameter, the media type will be returned along with the error ErrInvalidMediaParameter.
```
and we're not passing any optional parameters, it is safe for now. But it would be safe to check the error in case we do pass parameters in the future or the error use in `mime` changes.

Reply:
Same as above, not being an API change will do in 2.0.1